### PR TITLE
Add additional examples for first/last and take.

### PR DESCRIPTION
**Pull request by markstory**
It isn't always clear that the collection methods are on ResultSet. Give examples of methods people will commonly want.

### CODE REVIEW

**Comment by volponi** [comment]
Great! It would be nice if, somewhere in this page, explain that the resultset inherit the behaviors of Collection, and so it's possible to use any of Collections methods.

File: en/orm/retrieving-data-and-resultsets.rst [hunk 1]
```diff
@@ -732,8 +732,30 @@ articles quite easily::
...
+
+Getting an Arbitrary Index From a ResultSet
+-------------------------------------------
+
+You can use ``take()`` to get an arbitrary record from a ResultSet::
+
+    $result = $articles->find('all')->all();
+
+    // Get the 5th record
+    $row = $result->take(1, 4);
```
Comment (lorenzo) [inline en/orm/retrieving-data-and-resultsets.rst]: Can you change this to `->skip(4)->first()` ?

**Comment by lorenzo** [comment]
@volponi That is explained here http://book.cakephp.org/3.0/en/orm/query-builder.html#queries-are-collection-objects

**Comment by volponi** [comment]
@lorenzo right, this info is important when you're trying to build the query. But not important when using the resultset? As a newbie user, the Query problem is solved by CakePHP ORM beautifully. The retrieving part, without information, were not. 

Please note that this is my personal experience with the documentation right now, as I was trying to use CakePHP 3.0 for the first time (have some experience using 2.x).

Also, thank you guys for making this incredible software :+1:

**Comment by lorenzo** [comment]
@volponi It is important to know, the thing is that it is difficult to find a place for that info. Maybe repeating it would solve the problem?

**Comment by volponi** [comment]
@lorenzo repeating and linking to the Collection helper. That would clarify for sure.

**Comment by markstory** [comment]
@lorenzo Fixed, and I added a line about resultsets being collections as well.

**Pull request merged**